src/Fantomas.Core/SyntaxOak.fs

```diff
@@ -766,49 +766,64 @@ type RecordFieldNode(fieldName: IdentListNode, equals: SingleTextNode, expr: Exp
     member val Equals = equals
     member val Expr = expr
 
-type ExprRecordNode
+[<AbstractClass>]
+type ExprRecordNode(openingBrace: SingleTextNode, fields: RecordFieldNode list, closingBrace: SingleTextNode, range) =
+    inherit NodeBase(range)
+
+    member val OpeningBrace = openingBrace
+    member val Fields = fields
+    member val ClosingBrace = closingBrace
+    member x.HasFields = List.isNotEmpty x.Fields
+
+type ExprCopyableRecordNode
```

Comment:
I'm not sure `Copyable` really should be reflected in this name.
All records are copyable, so I really think you can call this one `ExprRegularRecordNode`.
The copy expression is not always there, so I wouldn't let it be a part of the name.